lib/handler.js

```diff
@@ -0,0 +1,15 @@
+import {errors} from '../errors/index'
```

Comment:
this will not work in production. You need to use `require` and create the url using `process.env.PWD`